massa-models/src/endorsement.rs

```diff
@@ -197,6 +200,43 @@ impl DeserializeCompact for Endorsement {
     }
 }
 
+// HERE
+// pub struct ShortEndorsement {
+//     pub sender_pub_key: PublicKey,
+//     pub signature: Signature,
+// }
+
+// HERE
+impl DeserializeCompactV2 for Endorsement {
+    fn from_bytes_compact_v2<'a>(buffer: &'a [u8]) -> IResult<&[u8], Self> {
+        tuple((
```

Comment:
I think we could write a function like macro for this tuple + map.

Reply:
Yeah we could but we need to have a meeting about it. I think it's very important to be clear about what it allows and what it doesn't and for what reasons.